templates/tsig_keys.j2

```diff
@@ -0,0 +1,7 @@
+// {{ ansible_managed }}
```

Comment:
And you should replace this row with ``{{ ansible_managed | comment('c') }}``
This is the way to go to enable multi-line ``ansible_managed``

Reply:
``nsupdate`` will delete this lines anyway...